provisioning/meta/playbooks/transactor-setup.yml

```diff
@@ -0,0 +1,20 @@
+---
+- hosts: transactor-provisioning
+  tasks:
+  - name: update system
+    apt: upgrade=yes
+  - name: setup git ppa repo
+    apt_repository: repo=ppa:git-core/ppa
+  - name: install git
+    apt: name=git state=present update_cache=true
+  - name: checkout mediachain
+    git: repo=https://github.com/mediachain/mediachain.git dest=/home/ubuntu/mediachain
+  - name: run setup script
+    shell: /home/ubuntu/mediachain/scripts/ubuntu-setup.sh && touch /home/ubuntu/.ubuntu-setup
```

Comment:
let's adapt this to use ansible instead of being a shell script. ansible is idempotent and offers a lot of flexibility